.reduxrc

```diff
@@ -0,0 +1,8 @@
+{
+  "sourceBase":"src",
+  "testBase":"tests",
+  "smartPath":"containers",
+  "dumbPath":"components",
+  "fileCasing":"pascal"
```

Comment:
If you use the default setting, what keyword do you use in the templates to interpolate the filename? Right now it's, for example, `<%= pascalEntityName %>`, is there something like `<%= defaultCase %>`?